e2e/.gitignore

```diff
@@ -6,6 +6,9 @@ tsc-out/
 out/
 
 puppeteer-access-token.txt
+reader-puppeteer-access-token.txt
```

Comment:
nit: maybe stick these into an ignored subdirectory, worried about these filenames going out of sync, then accidentally checking one of these tokens in

e.g. e2e/impersonation-tokens/ or e2e/signin-tokens/